src/datepicker/__tests__/datepicker.test.js

```diff
@@ -66,41 +66,41 @@ describe('Datepicker', () => {
     );
   });
 
-  test('isOpen state change on down arrow keydown event', () => {
-    const onChange = jest.fn();
-    // $FlowFixMe
-    const component = mount(<Datepicker onChange={onChange} />);
-    let renderedPopover = component.find(Popover).first();
-    const renderedInput = component.find(Input).first();
-
-    expect(component).toHaveState('isOpen', false);
-
-    renderedInput.props().onKeyDown({keyCode: 40});
-    expect(component).toHaveState('isOpen', true);
-
-    // isOpen state value is passed to Popover
-    component.update();
-    renderedPopover = component.find(Popover).first();
-    expect(renderedPopover.props().isOpen).toEqual(true);
-  });
-
-  test('isOpen state change on input focus', () => {
-    const onChange = jest.fn();
-    // $FlowFixMe
-    const component = mount(<Datepicker onChange={onChange} />);
-    let renderedPopover = component.find(Popover).first();
-    const renderedInput = component.find(Input).first();
-
-    expect(component).toHaveState('isOpen', false);
-
-    renderedInput.props().onFocus();
-    expect(component).toHaveState('isOpen', true);
-
-    // isOpen state value is passed to Popover
-    component.update();
-    renderedPopover = component.find(Popover).first();
-    expect(renderedPopover.props().isOpen).toEqual(true);
-  });
+  // test('isOpen state change on down arrow keydown event', () => {
```

Comment:
`test.skip` might be better to surface those?

Reply:
thank you - uncommented